backend/app/views/spree/admin/shared/_order_summary.html.erb

```diff
@@ -1,7 +1,15 @@
 <header id="order_tab_summary" data-hook>
   <dl class="additional-info">
     <dt class="order-state" id="order_status" data-hook><%= Spree.t(:status) %>:</dt>
-    <dd class="order-state"><span class="state <%= @order.state %>"><%= Spree.t(@order.state, scope: :order_state) %></span></dd>
+    <dd class="order-state">
+      <span class="pill pill-<%= @order.state %>">
+        <%= Spree.t(
+          @order.state,
+          scope: :order_state,
+          default: @order.state.humanize
```

Comment:
I'd rather not use `humanize` as a default. We want to see where we are missing translations (and avoid the cost of calling the inflector)